apps/console/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": ["./node_modules/@latitude-data/eslint-config/library.js"],
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-constant-condition": "off"
+  }
+}
```

Comment:
maybe move this new package to tools folder? I would like to keep the apps folder only for apps that are required for latitude to run.

Reply:
Noted, I'll move in another PR +1